portfolio/src/main/webapp/hobbies.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="UTF-8">
     <title>Hobbies</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBugz4DtpzusWF87Ebrq19ED5t0qh_PRG8"></script>
     <script src="script.js"></script>
   </head>
-  <body onload="makeCollapsible()">
+  <body onload="makeCollapsible(); createMap()">
```

Comment:
nit:  I would advice adding these into a single method so you don't clutter your body onload with more stuff.  Something like `init()` would work well